Content.Server/Strip/StrippableComponent.cs

```diff
@@ -9,13 +9,21 @@ namespace Content.Server.Strip
     [Access(typeof(StrippableSystem))]
     public sealed class StrippableComponent : SharedStrippableComponent
     {
+        /// <summary>
+        /// How long it takes to open the strip menu.
+        /// This should be relatively short so it's not a hassle
+        /// but so it also doesn't open immediately during melee combat
+        /// </summary>
         [ViewVariables]
         [DataField("openDelay")]
-        public float OpenDelay = 4f;
+        public float OpenDelay = 0.8f;
```

Comment:
With 0.8 this means as soon as you disarm someone you can start stripping all of their gear.